msdsales/read.go

```diff
@@ -3,38 +3,19 @@ package msdsales
 import (
 	"context"
 	"fmt"
-	"math"
 	"net/url"
 	"strings"
 
 	"github.com/amp-labs/connectors/common"
 )
 
-// TODO MS Sales allows fine control of reading
-// TODO basic read
-// $select = ReadParams{fields} [COMPLETED]
-// $expand = nested response
-// $orderby = list of fields with asc/desc keyword
-// TODO batch
-// $apply = batch
-// TODO search
-// $filter = query functions, comparisons
-// TODO pagination
-// $top = <int> of entries to return (ignored if header <Prefer: odata.maxpagesize>)
-// $count = counts all existing rows (@odata.count)
-
 var annotationsHeader = common.Header{
```

Comment:
Can you add a comment that adding this annotations header to the request will cause MS to return all available odata annotations in the response? (Just so someone reading this code in the future will understand the purpose of this header)